test/snippet/alphabet/concept_pre.cpp

```diff
@@ -0,0 +1,43 @@
+#include <seqan3/alphabet/concept_pre.hpp>
+#include <seqan3/alphabet/nucleotide/rna4.hpp>
+#include <seqan3/alphabet/structure/dot_bracket3.hpp>
+#include <seqan3/alphabet/structure/structured_rna.hpp>
+
+using namespace seqan3;
+
+int main()
+{
+
+{
+//! [include order]
+#include <seqan3/alphabet/concept_pre.hpp>
+
+// your custom alphabet
+
+#include <seqan3/alphabet/concept.hpp>
+//! [include order]
+}
+
+{
+using alphabet_type = rna4;
```

Comment:
Put this inside of the documentation block. I know it was probably not there before and you added it so that it compiles but it will also help the user.